runtime/testy/runtime.py

```diff
@@ -47,11 +47,14 @@ def runtime():
       print("Starting studentCode attempt: %s" % (restartCount,))
       spawnProcess(PROCESS_NAMES.STUDENT_CODE, runStudentCode)
       while True:
-        newBadThing = badThingsQueue.get(block=True)
-        print(RUNTIME_CONFIG.DEBUG_DELIMITER_STRING.value)
-        print(newBadThing)
-        if newBadThing.event in restartEvents:
-          break
+        try:
```

Comment:
This should be looked into more, to figure out the root cause. If no one understands *why* one approach works while the other doesn't, there's a good chance that the blocking issue will reappear when students start using the kit.